src/index.ts

```diff
@@ -6,30 +6,43 @@ function isEmptyObject(obj: unknown) {
   return typeof obj === 'object' && obj !== null && !Object.keys(obj).length;
 }
 
+export interface RemovalOptions {
```

Comment:
Having a default and secondary export in the same file gets really messy in CJS dist builds, causing types to get screwed up, do you mind either not exporting this or moving it to a separate file and exposing that in the `exports` config?

https://github.com/readmeio/remove-undefined-objects/blob/cbe07a73630a87ec95237e69d8449302c49de31d/package.json#L8-L14

We have a custom ESLint plugin that has the rule that's being flagged for this case below. There's some more details about why it can be problematic [here](https://github.com/readmeio/standards/blob/main/packages/eslint-plugin/docs/rules/no-dual-exports.md).

Reply:
👍 I removed the second export and filed https://github.com/readmeio/standards/issues/919. Thanks for the tip!